src/app/Services/validators.service.ts

```diff
@@ -0,0 +1,63 @@
+import { Injectable, inject } from '@angular/core';
+import { UserService } from './user.service';
+import { AbstractControl, ValidationErrors, ValidatorFn } from '@angular/forms';
```

Comment:
sequence of importing should be like following

1. Core imports
2. Third party imports
3. Application imports